app/routes/index.js

```diff
@@ -0,0 +1,22 @@
+var express = require('express');
+var router = express.Router();
+var formBuilder = require('../tools/formTool.js');
+
+var input1 = {field: 'input', type: 'text', id: 'firstName', label: 'First Name'};
```

Comment:
To make this project easier to use, I would suggest placing these objects into a JSON file and importing and parsing them. This will allow other developers to re-use the JSON file and output the desired elements automatically.